src/olympia/addons/decorators.py

```diff
@@ -7,12 +7,12 @@
 from olympia.addons.models import Addon
 
 
-def owner_or_unlisted_reviewer(request, addon):
+def owner_or_unlisted_viewer(request, addon):
```

Comment:
As above, the changes you're proposing would subtly change the meaning of this function: with your changes it reads to me as "the user needs to be an owner or an unlisted viewer" when it would actually do "the user needs to be an owner or an unlisted viewer or an unlisted reviewer (with full write access)".

Reply:
I renamed it to `owner_or_unlisted_viewer_or_reviewer` in eeec85729e.